cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -162,9 +162,12 @@
     })(window,document,'script','dataLayer','GTM-KMMLRS');</script>
     {# end Google Tag Manager #}
 
-    {# Customized Modernizr build that includes html5shiv.
-       Built via gulp-modernizer in `scripts.js` task. #}
-    <script src="{{ static('js/modernizr.min.js') }}"></script>
+    {# Customized Modernizr build that includes html5printshiv.
+       Built via gulp-modernizer in `scripts.js` task.
+       TODO: Determine if modernizr is actually needed! #}
+    <script>
+        {% include '/js/modernizr.min.js' %}
```

Comment:
How does it know where to find this file?

Reply:
Hmm I'm not sure why this would be problematic. It's how we always included common.js https://github.com/cfpb/cfgov-refresh/pull/3539/files#diff-89b43ee86bbd4952dc44cf751cfd775cL247

@rosskarchner Do you know?

Reply:
aren't you missing `static`?

Reply:
You don't need static unless you're making a request I thought. See line 248 and 261, we never used static for inlined JS before https://github.com/cfpb/cfgov-refresh/blob/2e125c5119d45f1ed0b72cf86a0de27cd119ac80/cfgov/jinja2/v1/_layouts/base.html#L248-L261

Reply:
Figured it out https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/test.py#L43-L53